### PR TITLE
Updating config schema fields and sdk version

### PR DESCRIPTION
**Pull request by mchavez**
- Updating config schema

### CODE REVIEW

File: pkg/connector/connector.go [hunk 1]
```diff
@@ -36,7 +37,7 @@ func (d *Connector) Asset(ctx context.Context, asset *v2.AssetRef) (string, io.R
 // Metadata returns metadata about the connector.
 func (d *Connector) Metadata(ctx context.Context) (*v2.ConnectorMetadata, error) {
 	return &v2.ConnectorMetadata{
-		DisplayName: "My Baton Connector",
+		DisplayName: "Baton Snowflake",
```
Comment (ggreer) [inline pkg/connector/connector.go:40]: 😆 oops.

Maybe we want to change the description too.

File: cmd/baton-snowflake/main.go [hunk 2]
```diff
@@ -5,41 +5,62 @@ import (
...
+)
+
+var (
+	AccountUrl           = field.StringField(snowflake.AccountUrl, field.WithRequired(true), field.WithDescription("Account URL."))
+	AccountIdentifier    = field.StringField(snowflake.AccountIdentifier, field.WithRequired(true), field.WithDescription("Account Identifier."))
+	UserIdentifier       = field.StringField(snowflake.UserIdentifier, field.WithRequired(true), field.WithDescription("User Identifier."))
+	PublicKeyFingerprint = field.StringField(snowflake.PublicKeyFingerPrint, field.WithRequired(true), field.WithDescription("Public Key Fingerprint."))
+	PrivateKeyPath       = field.StringField(snowflake.PrivateKeyPath, field.WithRequired(false), field.WithDescription("Private Key Path."))
+	PrivateKey           = field.StringField(snowflake.PrivateKey, field.WithRequired(false), field.WithDescription("Private Key (PEM format)."))
+	CacheDisabled        = field.StringField(batonCacheDisable, field.WithRequired(false), field.WithDescription("Verbose mode shows information about new memory allocation."))
```
Comment (ggreer) [inline cmd/baton-snowflake/main.go:35]: This isn't hooked up to anything, so we should delete it. Same goes for the other cache config settings.

Comment (mchavez) [reply cmd/baton-snowflake/main.go]: Fixed

File: pkg/connector/connector.go (hunk 1 above)
Comment (mchavez) [reply pkg/connector/connector.go:40]: Fixed

**Review by mgaeta — APPROVED** [review]

**Pull request merged**